### PR TITLE
Reviews

### PR DESCRIPTION
**Pull request by EmanueleMinotto**

### CODE REVIEW

File: src/Config/Node.php [hunk 1]
```diff
@@ -81,22 +82,22 @@ class Node
      */
     public function __construct(Application $app, $dbTable, $action)
     {
-        if (! isset($app['manager-config'])) {
```
Comment (Ocramius) [inline src/Config/Node.php]: These should be reverted

File: src/Config/Node.php [hunk 2]
```diff
@@ -114,17 +115,17 @@ public function __construct(Application $app, $dbTable, $action)
...
         }
 
-        $this->dbTable     = $dbTable;
-        $this->columns     = $tableConfig['columns'];
-        $this->order       = isset($tableConfig['order']) ? $tableConfig['order'] : 'DESC';
+        $this->dbTable = $dbTable;
+        $this->columns = $tableConfig['columns'];
+        $this->order = isset($tableConfig['order']) ? $tableConfig['order'] : 'DESC';
         $this->orderColumn = isset($tableConfig['orderColumn']) ? $tableConfig['orderColumn'] : 'id';
-        $this->primaryKey  = isset($tableConfig['pk']) ? $tableConfig['pk'] : 'id';
```
Comment (Ocramius) [inline src/Config/Node.php]: These alignment changes should be reverted

File: src/Config/Node.php [hunk 3]
```diff
@@ -208,7 +209,6 @@ public function getItemPerPage()
     }
 
     /**
-     * @return null
```
Comment (Ocramius) [inline src/Config/Node.php]: `@return void`

File: src/Controller/ManagerControllerProvider.php [hunk 4]
```diff
@@ -43,7 +44,7 @@ public function __construct(AdapterInterface $adapter)
     }
 
     /**
-     * {@inheritDoc}
+     * {@inheritdoc}
```
Comment (Ocramius) [inline src/Controller/ManagerControllerProvider.php]: It should be `inheritDoc`: revert 

Also, disable IDE autoformatting :-P

File: src/Controller/ManagerControllerProvider.php [hunk 5]
```diff
@@ -70,8 +71,8 @@ public function connect(Application $app)
      */
     public function index(Application $app, Request $request, $dbTable, $page)
     {
-        $config     = new Node($app, $dbTable, 'index');
-        $columns    = $config->getColumns();
+        $config = new Node($app, $dbTable, 'index');
```
Comment (Ocramius) [inline src/Controller/ManagerControllerProvider.php]: Revert alignment changes

File: src/Controller/ManagerControllerProvider.php [hunk 6]
```diff
@@ -90,24 +91,24 @@ public function index(Application $app, Request $request, $dbTable, $page)
         foreach ($result as $key => $row) {
             foreach ($columns as $name => $value) {
                 if (isset($action['modifier'][$name])) {
-                    $callable            = $action['modifier'][$name];
```
Comment (Ocramius) [inline src/Controller/ManagerControllerProvider.php]: Revert

File: src/Controller/ManagerControllerProvider.php [hunk 7]
```diff
@@ -90,24 +91,24 @@ public function index(Application $app, Request $request, $dbTable, $page)
...
 
         return $app['twig']->render($app['manager-config']['view']['index'], [
-            'rows'         => $result,
-            'title'        => $columns,
-            'action'       => $config->getAction(),
-            'header'       => $config->getHeader(),
-            'icon'         => $config->getIcon(),
-            'total'        => $total,
-            'pages'        => $pages,
-            'pagination'   => $pagination,
```
Comment (Ocramius) [inline src/Controller/ManagerControllerProvider.php]: Revert

File: src/Db/Adapter/AdapterInterface.php [hunk 8]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (Ocramius) [inline src/Db/Adapter/AdapterInterface.php]: Revert

File: src/Db/Adapter/PdoAdapter.php [hunk 9]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (Ocramius) [inline src/Db/Adapter/PdoAdapter.php]: Revert

File: tests/Db/Adapter/PdoAdapterTest.php [hunk 10]
```diff
@@ -228,14 +229,14 @@ public function testShouldFetchConfig()
...
-        $result  = $adapter->fetchByConfig($config, $pagination = 1);
+        $result = $adapter->fetchByConfig($config, $pagination = 1);
 
         $this->assertSame(1, $result);
     }
 
     public function testShouldFetchConfigUsingSpecifiedQuery()
     {
-        /** @var \Manager\Config\Node|\PHPUnit_Framework_MockObject_MockObject $pdo */
+        /* @var \Manager\Config\Node|\PHPUnit_Framework_MockObject_MockObject $pdo */
```
Comment (Ocramius) [inline tests/Db/Adapter/PdoAdapterTest.php]: These are actually the only correct styling changes

Comment (EmanueleMinotto) [inline tests/Db/Adapter/PdoAdapterTest.php]: lol, this is the second problem discovered in php-cs-fixer (after https://github.com/pennyphp/penny/issues/64) using php-cs-fixer

**Pull request closed without merging**